### PR TITLE
Updates to topic metadata - 2019.09.10

### PR DESCRIPTION
**Pull request by IanHakes-okta**
## Description:
- **What's changed?** 
Updated title, added metadata
- **Is this PR related to a Monolith release?** 
No

### Resolves:

* [OKTA-288174](https://oktainc.atlassian.net/browse/OKTA-288174)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/concepts/authentication/index.md [hunk 1]
```diff
@@ -15,12 +15,7 @@ The sign-in experience is one of the most important user experiences any app dev
 
 Okta provides many options for developers to handle the authentication experience that are built on top of the underlying feature set in Okta. Password policies, strong and adaptive authentication policies, and password reset workflow can all be configured easily in the administrator UI. Many of these policies can also be controlled through Okta's API.
 
-Okta provides the OAuth 2.0 and OpenID Connect implementations, the Authentication API, the Sign-in Widget, and the Auth JS SDK:
-
-* [OAuth 2.0, OpenID Connect, and Social Login](#oauth-20-openid-connect-and-social-login)
-* [Authentication API](#authentication-api)
-* [Sign-In Widget](#sign-in-widget)
-* [Auth SDK](#auth-sdk)
+Okta provides OAuth 2.0 and OpenID Connect implementations, the Sign-in Widget, the Auth JS SDK, and the Authentication API.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/concepts/authentication/index.md]: I believe that is: Sign-In Widget

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/index.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Add an external Identity Provider
+meta:
+  - name: description
+    content: Okta supports authentication with an external OpenID Connect and SAML Identity Providers. Get an overview of the process and prerequisites, and the instructions required to set one up.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/index.md]: Okta supports authentication with external OpenID Connect Identity Providers as well as SAML. Get an overview of the process and prerequisites, and the instructions required to set one up.

File: packages/@okta/vuepress-site/docs/guides/set-up-event-hook/index.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 ---
 title: Set Up Event Hooks
-excerpt: Configure and enable an Event Hook
+meta:
+  - name: description
+    content: Event hooks are outbound calls from Okta, sent when specified events occur in your Okta organization. Get information on eligible hooks and how to set up event hooks in this guide.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/set-up-event-hook/index.md]: Might have @motikagan-okta take a look at this one, just for sanity sake.

File: packages/@okta/vuepress-site/docs/reference/webfinger/index.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 title: WebFinger
 meta:
   - name: description
-    content: The Webfinger protocol helps to determine the identity provider where a given username should be routed. This guide explains the process of finding a user's identity provider.
+    content: The Webfinger protocol helps to determine the identity provider where a given username should be routed. This guide explains the process of finding the identity provider for a user.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/webfinger/index.md]: style is to capitalize Identity Provider:

The Webfinger protocol helps to determine the Identity Provider where a given username should be routed. This guide explains the process of finding the Identity Provider for a user.

File: packages/@okta/vuepress-site/docs/concepts/authentication/index.md (hunk 1 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/concepts/authentication/index.md]: Fixed

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/index.md (hunk 2 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/add-an-external-idp/index.md]: Good catch - thanks!

File: packages/@okta/vuepress-site/docs/reference/webfinger/index.md (hunk 4 above)
Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/reference/webfinger/index.md]: Fixed

File: packages/@okta/vuepress-site/docs/guides/set-up-event-hook/index.md (hunk 3 above)
Comment (motikagan-okta) [reply packages/@okta/vuepress-site/docs/guides/set-up-event-hook/index.md]: Lgtm

**Pull request merged**